src/util/firebase.js

```diff
@@ -1,5 +1,6 @@
 import { initializeApp } from "firebase/app";
 import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore"; // will be used later
```

Comment:
this line should be deleted, firesore is imported on line 8, this is created ESLint error

Reply:
also run `npm run format` to pass prettier test

Reply:
Fixed and resolved!